Makefile

```diff
@@ -11,7 +11,11 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-.PHONY: all gobuild static docker release certs test clean netkitten test-registry run-functional-tests gremlin benchmark-test gogenerate run-integ-tests image-cleanup-test-images
+PAUSE_CONTAINER_IMAGE = "amazon/amazon-ecs-pause"
+PAUSE_CONTAINER_TAG = "0.1.0"
```

Comment:
I was asking about the logic of the code where we need to create a container using this image, apologies for the lack of clarity earlier.

To rephrase, how are you planning to surface this tag information to the Agent when it has to create a container using this image? Should we always use a constant tag (Example: `latest`)? Or should that be in code somewhere (which means we update with every release of pause container)? What are the pros/cons for these?